src/plugins/console/public/application/models/sense_editor/sense_editor.ts

```diff
@@ -96,6 +96,11 @@ export class SenseEditor {
       return;
     }
 
+    if (parsedReq.data.some((doc) => utils.hasComments(doc))) {
+      this.coreEditor.autoIndent(parsedReq.range);
```

Comment:
Can we add a comment here to briefly explain why the presence of comments requires us to delegate indentation to `coreEditor` instead of encapsulating that logic here?

Reply:
Added